lib/dangermattic/plugins/manifest_pr_checker.rb

```diff
@@ -99,6 +99,23 @@ def check_swift_package_resolved_updated_strict(manifest_path:, manifest_lock_pa
       )
     end
 
+    # Check for duplicate Package.resolved files across all provided SwiftPM package directories
+    #
+    # @param report_type [Symbol] (optional) The type of report for the message. Types: :error, :warning (default), :message.
+    # @param spm_root_paths [Array<String>] Array of paths to SwiftPM package root directories (e.g. ['Modules'])
```

Comment:
I wonder if we couldn't auto-detect those though, by globbing the repo for all `Package.swift` files and `xcodeproj`/`xcworkspace` files 🤔

Reply:
Yeah that was my first thought. I considered it would make it a bit harder to find the `Package.resolved` file(s) corresponding to a given `Package.swift` (though we could also just see if the overall number matches 🤔), but maybe I'm missing something from the possible scenarios with `xcodeproj` / `xcworkspace` 🤔

Reply:
I'm thinking of going on the route of checking for matching `Package.swift` and `Package.resolved`, without a list of paths 🤔 seems simple enough and will solve the issue.

Reply:
Pushed another implementation just comparing the count and globbing the repo (removing the list param) at 7ca1da561eb090434f87a891f3b039984d7296dc